### PR TITLE
SOFTWARE-3359 (The multiuser tests needs globus proxy utils and the first test to work)

### PR DESCRIPTION
**Pull request by efajardo**
Pull request to fix errors on the last [PR](https://github.com/opensciencegrid/osg-test/pull/149) for [SOFTWARE-3359](https://opensciencegrid.atlassian.net/browse/SOFTWARE-3359)

### CODE REVIEW

**Comment by efajardo** [comment]
@brianhlin I made the changes you requested. can you please check again?

File: osgtest/tests/test_450_xrootd.py [hunk 1]
```diff
@@ -33,18 +33,18 @@ def test_01_xrdcp_local_to_server(self):
...
+            core.state['xrootd.copied-to-server'] = True
         self.assertEqual(status, 0, fail)
         self.assert_(file_copied, 'Copied file missing')
 
     def test_02_xrootd_multiuser(self):
-        core.skip_ok_unless_installed('xrootd', 'xrootd-client', 'xrootd-multiuser', by_dependency=True)
-        temp_dir = "/tmp/vdttest"
-        if core.config['xrootd.multiuser'] == "ON":
+        core.skip_ok_unless_installed('xrootd', 'xrootd-client', 'globus-proxy-utils', 'xrootd-multiuser', by_dependency=True)
+        self.skip_bad_unless(core.state['xrootd.copied-to-server'] is True, 'File to check ownership does not exist')
```
Comment (brianhlin) [inline osgtest/tests/test_450_xrootd.py]: Style nitpick:

```suggestion
        self.skip_bad_unless(core.state['xrootd.copied-to-server'], 'File to check ownership does not exist')
```

File: osgtest/tests/test_150_xrootd.py [hunk 2]
```diff
@@ -76,7 +76,7 @@ def test_02_configure_hdfs(self):
         files.append(core.config['xrootd.config'], hdfs_config, backup=False)
 
     def test_03_configure_multiuser(self):
-        core.skip_ok_unless_installed('xrootd-multiuser')
+        core.skip_ok_unless_installed('xrootd-multiuser','globus-proxy-utils', by_dependency=True)
         core.config['xrootd.multiuser'] = "ON"
```
Comment (brianhlin) [inline osgtest/tests/test_150_xrootd.py]: We should initially set this to `False` and then `True` at the very end of the test

File: osgtest/tests/test_450_xrootd.py [hunk 3]
```diff
@@ -33,18 +33,18 @@ def test_01_xrdcp_local_to_server(self):
...
+        temp_dir = core.config['xrootd.tmp-dir']
```
Comment (brianhlin) [inline osgtest/tests/test_450_xrootd.py]: You only use `temp_dir` once in this function, you may as well get rid of this line and replace `temp_dir` with `core.config['xrootd.tmp-dir']`

File: osgtest/tests/test_450_xrootd.py [hunk 4]
```diff
@@ -33,18 +33,18 @@ def test_01_xrdcp_local_to_server(self):
...
+        if core.config['xrootd.multiuser'] == "ON" and core.state['xrootd.copied-to-server']:
```
Comment (brianhlin) [inline osgtest/tests/test_450_xrootd.py]: The first part of this if block should be replaced with `self.skip_bad_unless(core.config['xrootd.multiuser'], "<fail text here>")`. The second part is superfluous with your `self.skip_bad_unless(core.state['xrootd.copied-to-server']` line

File: osgtest/tests/test_450_xrootd.py [hunk 5]
```diff
@@ -33,18 +33,18 @@ def test_01_xrdcp_local_to_server(self):
...
             file_path = os.path.join(temp_dir, 'copied_file.txt')
             result_perm = core.check_file_ownership(file_path, core.options.username)
```
Comment (brianhlin) [inline osgtest/tests/test_450_xrootd.py]: Same thing about `result_perm` as `temp_dir` being only used once above

File: osgtest/tests/test_840_xrootd.py [hunk 6]
```diff
@@ -11,7 +11,8 @@ def test_01_stop_xrootd(self):
             files.restore('/etc/xrootd/auth_file', "xrootd")
             if not core.rpm_is_installed('xrootd-lcmaps'):
                 files.restore('/etc/grid-security/xrd/xrdmapfile', "xrootd")
-
```
Comment (brianhlin) [inline osgtest/tests/test_840_xrootd.py]: Errant whitespace change

**Pull request merged**